apis/apis.json

```diff
@@ -226,13 +226,15 @@
     "serviceYaml": "containeranalysis_v1.yaml",
     "productName": "Google Container Analysis",
     "productUrl": "https://cloud.google.com/container-registry/docs/container-analysis/",
-    "version": "1.0.0-beta05",
+    "version": "1.0.0",
     "type": "grpc",
     "description": "Recommended Google client library to access the Google Cloud Container Analysis API.",
     "tags": [ "analysis", "artifact", "metadata", "grafeas" ],
     "dependencies": {
-      "Google.Cloud.Iam.V1": "1.2.0",
-      "Grafeas.V1": "project"
+      "Google.Api.Gax.Grpc": "2.10.0",
+      "Google.Cloud.Iam.V1": "1.3.0",
+      "Grafeas.V1": "project",
```

Comment:
Just flagging in case it's a problem, but this is not a explicti version dependency, but to the project, and the release is not a beta.

Reply:
That's okay because it's being released at the same time. (It would fail otherwise.)